src/ring_theory/ideal/basic.lean

```diff
@@ -400,6 +400,7 @@ variables [comm_ring α] (I : ideal α) {a b : α}
 -- Note that at present `ideal` means a left-ideal,
 -- so this quotient is only useful in a commutative ring.
 -- We should develop quotients by two-sided ideals as well.
+@[reducible]
```

Comment:
can you add a comment here also explaning why you want this to be reducible?

Reply:
I have added it to the docstring.